test/unit/annotation_spec.js

```diff
@@ -1630,6 +1656,62 @@ describe("annotation", function () {
       }, done.fail);
     });
 
+    it("should render annotation for printing", function (done) {
```

Comment:
This unit test should be named `should render checkboxes for printing` to make the distinction with radio buttons for a later patch, and it should be moved up just above the radio button unit test above (line 1625) so that the checkbox tests are grouped.